repose-aggregator/commons/utilities/src/test/scala/org/openrepose/commons/utils/servlet/http/HttpServletRequestWrapperTest.scala

```diff
@@ -93,6 +94,7 @@ class HttpServletRequestWrapperTest extends FunSpec with BeforeAndAfter with Mat
       wrappedRequest.getHeaderNames.asScala.toList should contain theSameElementsAs headerMap.keys.filterNot( _ == "foo")
     }
 
+    //todo: fix this test -- it should not be case sensitive
```

Comment:
Nope, not twice, once with a different case key. As for keeping the original... I don't know that it matters to me. If you're explicitly passing a differently cased header to addHeader, I would lean toward using that since it was the last change made, and it was made explicitly.

Yeah, I wasn't thrilled about the equalsIgnoreCase things. There is probably a better way to do that which leveraged the Ordering, but I'll have to look at it when I get to the office.